src/systemdhook.c

```diff
@@ -815,20 +806,23 @@ int main(int argc, char *argv[])
 	}
 	char *id = YAJL_GET_STRING(v_id);
 
-	/* 'bundlePath' must be specified for the OCI hooks, and from there we read the configuration file */
-	const char *bundle_path[] = { "bundlePath", (const char *)0 };
+	/* 'bundle' must be specified for the OCI hooks, and from there we read the configuration file */
```

Comment:
Can you leave a comment here explaining why we're looking for bundle or bundlePath?